tests/phpunit/Logger/JsonLoggerTest.php

```diff
@@ -246,12 +252,13 @@ public function metricsProvider(): iterable
 
         yield 'Non UTF-8 characters' => [
             false,
-            $this->createNonUtf8CharactersCalculator(),
+            new MetricsCalculator(2),
+            $this->createNonUtf8CharactersCollector(),
             [
                 'stats' => [
-                    'totalMutantsCount' => 1,
```

Comment:
same question, why was it changed? (I mean probably it's ok, I just didn't get the change)

Reply:
It tests against empty `MetricsCalculator`, and it does that because it's a test of an edge-case of `JsonLogger`. We do not actually care about these numbers here.